RELEASE_NOTES.md

```diff
@@ -1,5 +1,77 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Infra Client 15.5
+
+## New Cookbook Helpers
+
+Chef Infra Client now includes a new `chef-utils` gem, which ships with a large number of helpers to make writing cookbooks easier. Many of these helpers existed previously in the chef-sugar gem. We've renamed many of the named for consistency, while providing backwards compatibility with existing chef-sugar names. Existing cookbooks written with chef-sugar should work unmodified with any of these new helpers. Expect a Cookstyle rule in the near future to help you update existing chef-sugar code to use the newer built-in helpers.
+
+For more information all all of the new helpers available see the [chef-utils readme](https://github.com/chef/chef/blob/master/chef-utils/README.md)
+
+## Chefignore Improvements
+
+We've reworked how chefignore files are handled in knife, which has allowed us to close out a large number of long outstanding bugs. knife will now traverse all the way up the directory structure looking for a chefignore file. This means you can place a chefignore file in each cookkbook or any parent directory in your repository structure. Additionally we've made fixes that ensure that commmands like `knife diff` and `knife cookbook upload` always honor your chefignore files.
```

Comment:
Should we be applying `formatting` to the _knife_ mentions in this release note? I'm forgetting at this rate. See this paragraph and the paragraph in ln 17.

Reply:
Done back to 15.0